src/azure-cli/azure/cli/command_modules/policyinsights/_help.py

```diff
@@ -290,3 +290,56 @@
     text: >
         az policy metadata show --name ACF1000
 """
+
+helps['policy attestation'] = """
+type: group
+short-summary: Manage resource policy attestations.
+"""
+
+helps['policy attestation create'] = """
+type: command
+short-summary: Creates a policy attestation.
```

Comment:
Please use the first-person voice in the help message
```suggestion
short-summary: Create a policy attestation.
```